projects/packages/forms/src/class-wpcom-rest-api-v2-endpoint-forms.php

```diff
@@ -314,7 +314,7 @@ public function get_responses_permission_check() {
 			return $site_id;
 		}
 
-		if ( ! current_user_can( 'manage_options' ) || ! is_user_member_of_blog( get_current_user_id(), $site_id ) ) {
+		if ( ! current_user_can( 'edit_pages' ) ) {
```

Comment:
This update matches the behavior of the list page view:
https://github.com/Automattic/jetpack/blob/ca5ff93ae6d54977a25434dd8afb2f62ab9a45ed/projects/packages/forms/src/contact-form/class-contact-form-plugin.php#L500-L518
https://github.com/Automattic/jetpack/blob/653476e520d896d175fe995e8f190d8403795a83/projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php#L743-L761